File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F02.P1.java
class Main {
    static int probe(boolean cond) {
        // BEGIN PROBE F02.P1
        int v;
        if (cond) v = 5; else v = 9;
        return v;
        // END PROBE F02.P1
    }

    public static void main(String[] args) {
        System.out.println(probe(true));
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F02.P1.go
package main

import "fmt"

func probe(cond bool) int32 {
	// BEGIN PROBE F02.P1
	var v int32
	if cond {
		v = 5
	} else {
		v = 9
	}
	return v
	// END PROBE F02.P1
}

func main() {
	fmt.Println(probe(true))
}
